### PR TITLE
Included security details

### PR DESCRIPTION
**Pull request by chrislittle**
Included security details to guide customers on CLI commands passing over HTTPS and a note on storing credentials on local machine.  feedback welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,6 +318,9 @@ instructions below may not work properly on Windows.
 
 * If you want to make an executable, simply run `./scripts/build`. The binary will appear in the `./out` folder.
 
+## Security
```
Comment (ldmberman) [inline README.md]: I would rather say:

"
The CenturyLink Cloud Go CLI leverages our [public API](//www.ctl.io/api-docs/v2/) that serves all requests over HTTPS. Therefore credentials are encrypted when being transfered. Credentials stored on the local machine in the [config.yml](#Set-up-the-configuration-file) are not encrypted during installation and you are encouraged to use industry standard encryption tools in order to provide additional protection for them.
"

If it does not sound reasonable to you, we could use your version - it's ok.

Comment (chrislittle) [inline README.md]: i'm fine with that text.  do you want to modify it yourself?  and just close this pull?

Comment (ldmberman) [inline README.md]: Ok, I'll add it myself then

Comment (ldmberman) [inline README.md]: Done - 26e2df518932ca94085433cd16337d0892f5e05b

**Pull request closed without merging**